### PR TITLE
Updated dependencies

### PR DESCRIPTION
**Pull request by RebeccaStevens**
Based off of #213 

Closes #213

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,4 +2,3 @@ language: node_js
 node_js:
```
Comment (TehShrike) [inline .travis.yml:2]: may as well add 14 in I suppose
```suggestion
node_js:
  - "14"
```

File: package.json [hunk 2]
```diff
@@ -9,35 +10,41 @@
...
+    "rollup": "^2.33.1",
+    "tape": "^5.0.1",
+    "terser": "^5.5.1",
+    "ts-node": "^9.0.0",
+    "typescript": "^4.1.2"
   },
-  "license": "MIT",
-  "dependencies": {}
+  "engines": {
+    "node": ">=12"
```
Comment (TehShrike) [inline package.json:48]: node 10 is still in maintenance mode, was there some other reason to specify 12+?
```suggestion
    "node": ">=10"
```

**Comment by TehShrike** [comment]
So this PR adds TypeScript dependencies, but doesn't do anything with them yet?

**Comment by RebeccaStevens** [comment]
> So this PR adds TypeScript dependencies, but doesn't do anything with them yet?

This project already has "typescript" and "ts-node" as dev-dependencies; it uses them for testing for typescript.
This PR updates those dependencies, it doesn't add any new ones.

**Pull request merged**